pyproject.toml

```diff
@@ -6,10 +6,11 @@ license = "BSD-3"
 readme = "README.md"
 packages = []
 authors  = ["MIT ODL"]
-
+requires-poetry = ">2.1,<3"
```

Comment:
The `Dockerfile` specifies 1.7.1, so this causes the build to fail. No issues updating the Dockerfile to specify 2.1.3, though. It will also need `--no-root` when installing.